include/openenclave/enclave.h

```diff
@@ -43,6 +43,13 @@ OE_EXTERNC_BEGIN
 #define __thread
 #endif
 
+/** Continue calling next vectored exception handler */
```

Comment:
Consider being a bit more verbose here as these will end up in the doxygen comments for the enclave.h file on [openenclave.io](https://openenclave.io/apidocs/v0.8/enclave_8h.html). e.g. 
```
/**
 * Return value used by an enclave vectored exception handler to indicate
 * to the dispatcher that it should continue searching for the next handler.
 */
```

Reply:
Updated